### PR TITLE
Updated docs for settings for v7.26

### PR DESCRIPTION
**Pull request by Nielio**
Updated docs for settings for v7.26. Fixed order and add more headlines for matching the postman ui.
Also updated the screenshot.

### CODE REVIEW

File: src/pages/docs/postman/launching-postman/settings.md [hunk 1]
```diff
@@ -35,35 +35,53 @@ In the header toolbar of the Postman app, click the wrench icon and select "Sett
...
+* **Location** Path to local directory containing postman files.
+* **Allow reading files outside working directory:** Set this option to ON if you want Postman to read files from
+  outside the Postman working directory. Postman also persists your file paths in form-data request and binary file bodies. To learn more about this feature, refer to the section [form-data](/docs/postman/sending-api-requests/requests/#form-data).
+
+### HEADERS
+
+* **Send no-cache header** (recommended): Sending a no-cache header makes sure you get the freshest response from your server.
+* **Send Postman Token header:** This is primarily used to bypass a bug in Chrome. If an XmlHttpRequest is pending and. another request is sent with the same parameters then Chrome returns the same response for both of them. Sending a random token avoids this issue. This can also help you distinguish between request on the server side.
+* **Retain headers when clicking on links:** If you click on a link in a response, Postman creates a new GET request with that URL. If you want to retain the headers that you set in the previous request set "ON" here. This is useful if you are accessing mainly protected resources.
```
Comment (SueSmith) [inline src/pages/docs/postman/launching-postman/settings.md:74]: If we could replace the quotes with bold for ui items thx ("ON" here)

File: src/pages/docs/postman/launching-postman/settings.md [hunk 2]
```diff
@@ -35,35 +35,53 @@ In the header toolbar of the Postman app, click the wrench icon and select "Sett
...
+* **Automatically follow redirects:** Prevent requests that return a 300-series response from being automatically redirected.
+* **Send anonymous usage data to Postman:** Option to disable sending basic anonymous usage data (button clicks and app events) to Postman. Postman uses usage data to make a better product.
```
Comment (SueSmith) [inline src/pages/docs/postman/launching-postman/settings.md:76]: If we could alter the last sentence to something like "Postman feeds usage data into product improvements."

File: src/pages/docs/postman/launching-postman/settings.md [hunk 3]
```diff
@@ -35,35 +35,53 @@ In the header toolbar of the Postman app, click the wrench icon and select "Sett
...
+
+### USER INTERFACE
+
+* **Editor Font Size:** Adjust the font size in pixels for the text that appears in Postman. Note that this setting impacts only the *Test Scripts, Pre-request Scripts, and Response Pretty View*
```
Comment (SueSmith) [inline src/pages/docs/postman/launching-postman/settings.md:80]: If we could lose the italics here

File: src/pages/docs/postman/launching-postman/settings.md [hunk 4]
```diff
@@ -35,35 +35,53 @@ In the header toolbar of the Postman app, click the wrench icon and select "Sett
...
+* **Two-pane view**: Toggle between showing the response below, or beside, the request.
+* **Variable autocomplete:** Enable this to turn on autocomplete feature for your variables.
+
+[![general settings](https://i.imgur.com/fNq9yOe.png)](https://i.imgur.com/fNq9yOe.png)
```
Comment (SueSmith) [inline src/pages/docs/postman/launching-postman/settings.md:84]: the image is rendering a little pixelated for me - you could just remove the image update for now tbh

File: src/pages/docs/postman/launching-postman/settings.md [hunk 5]
```diff
@@ -35,35 +35,53 @@ In the header toolbar of the Postman app, click the wrench icon and select "Sett
 
 Postman tries to minimize the number of settings you have to change, so some defaults are automatically set. However, given the diversity of use cases, if you need to make adjustments, here's how:
 
+### REQUEST
```
Comment (SueSmith) [inline src/pages/docs/postman/launching-postman/settings.md:38]: I know these are upper case in the UI but can we just capitalize them i.e. Request etc?

File: src/pages/docs/postman/launching-postman/settings.md [hunk 6]
```diff
@@ -35,35 +35,53 @@ In the header toolbar of the Postman app, click the wrench icon and select "Sett
...
+
 * **Trim keys and values in request body:** If you’re using the form-data or url-encoded modes to send data to the server, switching this to "ON" will cause any parameters to be trimmed.
 * **New Code Generation Mode:** If you’re generating code snippets via the **Code** link, switching this to "ON" enables custom settings by programming language.
+* **Use next generation URL processing**
```
Comment (SueSmith) [inline src/pages/docs/postman/launching-postman/settings.md:42]: so we wouldn't normally link to a blog post in the body of a doc - could we change this to the docs link here? https://learning.postman.com/docs/postman/sending-api-requests/requests/#encoding-your-request-urls

**Comment by Nielio** [comment]
@SueSmith I did everything you want :D
I'm not a fan of very long lines, but if this is the style here, i am ok with it ;)

**Pull request merged**